ui/js/controllers/logviewer.js

```diff
@@ -158,17 +158,25 @@ logViewerApp.controller('LogviewerCtrl', [
                 if(artifactList.length > 0){
                     $scope.artifact = artifactList[0].blob;
 
-                    var revision = $scope.artifact.header.revision.substr(0,12);
-                    $scope.logRevisionFilterUrl = $scope.urlBasePath
-                        + "#/jobs?repo="
-                        + $scope.repoName + "&revision=" + revision;
-
-                    // Store the artifact epoch date string in a real date object for use
-                    var startTime = $scope.artifact.header.starttime;
-                    var startDate = new Date(0);
-                    startDate.setUTCSeconds(startTime);
-
-                    $scope.logDisplayDate = startDate.toString();
+                    $scope.logProperties = _.map(
```

Comment:
I wondered about `logHeaderProperties` or something to differentiate from other potential log properties, but this looks fine to me either way.